### PR TITLE
CentOS based VW build image

### PR DESCRIPTION
**Pull request by eisber**

### CODE REVIEW

File: vowpal_wabbit/centos7_6_1860-build.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   && ln -s /opt/cmake/bin/cmake /usr/bin/cmake \
+   && rm -f cmake-$version.$build-Linux-x86_64.sh
+
+# Install gcc-9.2
+RUN wget ftp://ftp.mirrorservice.org/sites/sourceware.org/pub/gcc/releases/gcc-9.2.0/gcc-9.2.0.tar.gz \
+   && tar xfz gcc-9.2.0.tar.gz \
+   && cd gcc-9.2.0 \
+   && ./contrib/download_prerequisites \
+   && ./configure --disable-multilib --enable-languages=c,c++ --with-pic \
+   && make -j 16 \
```
Comment (jackgerrits) [inline vowpal_wabbit/centos7_6_1860-build.Dockerfile]: Can you make this parallelization generic? I dont think it will work on a build machine like this

**Comment by gramhagen** [comment]
@jackgerrits Are we good to merge here?

File: vowpal_wabbit/centos7_6_1860-build.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# (replace 27c82a691b62 with whatever got printed at the end of docker build)
+# * docker tag 27c82a691b62 eisber/centos7.6.1810-build:0.1.0
+# * docker push eisber/centos7.6.1810-build
+
+FROM centos:centos7.6.1810
+
+RUN yum -y updateinfo \
+   && yum install -y \
+   # C++ to bootstrap
+   gcc-c++ gcc git make wget help2man \
```
Comment (jackgerrits) [inline vowpal_wabbit/centos7_6_1860-build.Dockerfile]: Unless you need help2man for something that is not needed

**Comment by jackgerrits** [comment]
I wasn't sure if the pipeline itself was ready, so I left this open if you needed to make changes

File: vowpal_wabbit/centos7_6_1860-build.Dockerfile (hunk 2 above)
Comment (jackgerrits) [reply vowpal_wabbit/centos7_6_1860-build.Dockerfile]: Checked with Markus, it is not

**Comment by jackgerrits** [comment]
It's ready to go, merging

**Pull request merged**